fixture-monkey-kotlin/src/main/kotlin/com/navercorp/fixturemonkey/kotlin/property/KPropertyProperty.kt

```diff
@@ -44,7 +44,8 @@ data class KPropertyProperty(
     override fun getValue(instance: Any): Any? {
         val javaMethod = this.kProperty.getter.javaMethod
 
-        if (javaMethod != null && Modifier.isPublic(javaMethod.modifiers)) {
+        if (javaMethod != null) {
+            this.kProperty.isAccessible = true
```

Comment:
is it necessary for `value class`?

Reply:
Duration has a rawValue for the StorageUnit, so we need to know some private method values to get the StorageUnit information.

```kotlin
private val storageUnit get() = if (isInNanos()) DurationUnit.NANOSECONDS else DurationUnit.MILLISECONDS
```

The get method of the StorageUnit is declared private, which means that it can't be called from outside. The code below is the format that would be added if you decompiled the compiled code from the class file again.

```kotlin
private final val storageUnit: kotlin.time.DurationUnit /* compiled code */
    private final get

```